README.md

```diff
@@ -5,47 +5,134 @@
+```python
+# Pixeltable is downloading the model from Huggingface (if necessary), instantiating it, and caching it for later use
+t['detections'] = huggingface.detr_for_object_detection(t.input_image, model_id='facebook/detr-resnet-50')
+```
+Learn more on how to use computed colums through the example of [comparing object detection models](https://pixeltable.readme.io/docs/object-detection-in-videos).
+
+### Build User-Defined Functions to expand Pixeltable
+```python
+# The longest_word Python function isn't a Pixeltable UDF (yet); Adding the decorator turns it into a UDF:
+@pxt.udf
+def longest_word(sentence: str, strip_punctuation: bool = False) -> str:
+    words = sentence.split()
+    if strip_punctuation:  # Remove non-alphanumeric characters from each word
+        words = [''.join(filter(str.isalnum, word)) for word in words]
+    i = np.argmax([len(word) for word in words])
+    return words[i]
+```
+Learn more on how to leverage [UDFs](https://pixeltable.readme.io/docs/user-defined-functions-udfs)
+
+### Automate data operations with Views
+```python
+# In this example, the view is defined by iteration over the chunks of a DocumentSplitter.
+chunks_t = pxt.create_view(
+    'rag_demo.chunks',
+    documents_t,
+    iterator=DocumentSplitter.create(
+        document=documents_t.document,
+        separators='token_limit', limit=300)
+)
+```
+Learn more on how to leverage Views to build your [RAG workflow](https://pixeltable.readme.io/docs/document-indexing-and-rag).
+
+### Evaluate models performance
+```python
+# The computation of the mAP metric can simply become query over the evaluation output, aggregated with the mean_ap() function.
+frames_view.select(mean_ap(frames_view.eval_yolox_tiny), mean_ap(frames_view.eval_yolox_m)).show()
+```
+Learn more on how to leverage Pixeltable for [Model analytics](https://pixeltable.readme.io/docs/object-detection-in-videos).
+
+### Working with inference APIs
+```python
+chat_t = pxt.create_table('together_demo.chat', {'input': pxt.StringType()})
+
+# The chat-completions API expects JSON-formatted input:
+messages = [{'role': 'user', 'content': chat_t.input}]
+
+# This example shows how additional parameters from the Together API can be used in Pixeltable to customize the model behavior.
+chat_t['output'] = chat_completions(
+    messages=messages,
+    model='mistralai/Mixtral-8x7B-Instruct-v0.1',
+    max_tokens=300,
+    stop=['\n'],
+    temperature=0.7,
+    top_p=0.9,
+    top_k=40,
+    repetition_penalty=1.1,
+    logprobs=1,
+    echo=True
+)
+chat_t['response'] = chat_t.output.choices[0].message.content
+
+# Start a conversation
+chat_t.insert([
+    {'input': 'How many species of felids have been classified?'},
+    {'input': 'Can you make me a coffee?'}
+])
+chat_t.select(chat_t.input, chat_t.response).head()
+```
+Learn more on how to leverage LLms such as [Together AI](https://pixeltable.readme.io/docs/together-ai) in Pixeltable.
```

Comment:
LLms -> LLMs

Together AI isn't an LLM though. Let's be accurate ("inference services such as Together AI" or "LLMs hosted by services such as Together AI" or some such)